### DISCUSSION TITLE
Schedule details

### DISCUSSION THREAD

**Discussion by orrious** [tagyoureit/nodejs-poolController#553] [Nixie]
@rstrouse could you share what your schedules looks like? 

After changing the circuits around yesterday, my Cleaner did not run this morning.   Cleaner circuit is back to Master Cleaner type so I have to run the Pool circuit for the Cleaner circuit to run.  So I set the schedule for them both to start at 8am.  The Pool circuit started but the Cleaner did not.

I assume you can't start multiple circuits at the same time?  How much time needs to be left in between?  Pool at 8:00am and Cleaner at 8:05?

**Comment by tagyoureit** [comment]
Do you have the cleaner delay on?  That could be one variable that would prevent the schedule from starting at the same time.  Also, if you set them for exactly the same time the app may pick up the cleaner schedule first.  Make sure you start your cleaner after the pump has finished priming, and after the cleaner delay and at least 1 minute after the pump starts.

**Comment by orrious** [comment]
@rstrouse @tagyoureit I would be really interested in seeing what your:  Schedules, Pump circuits / speeds, rem chem settings, look like.  

Do you run your filter pump 24/7?  What speeds during the day? What are your ORP and pH Setpoints

**Comment by rstrouse** [comment]
I do not run my filter pump 24/7.

My schedules are:
* 8am-10am:  Cleaner - This is plenty of time for my suction side cleaner to have its way with the pool.  (2,650 rpm)
* 10am-8pm: Pool -  General pool circuit run.  Provides just enough chlorine generation and allows REM Chem to keep balance (1,850)
* 2pm - 2:30pm: Spillway - Run the spillway torrent to put chlorinated water into the spa (2,800 rpm)

I do not use REM Chem to control my chlorinator.  This is because I have a huge pool and an IC40.  The original IC40 I had was capable of producing much more chlorine but the new one barely keeps up at 100% from 10am to 8pm.  The original is still running strong in a friend's pool and is 13 years old.  I swapped mine out since I had a new cell in my pool closet assuming the 13yo cell was soon to slip into Davey Jones locker.  I am sure that it about to fail at any time.

**Reply by tagyoureit** [reply]
This was my pump last night... so take my schedules with a grain of salt.  I'm also going to be changing these shortly.

* ~20 hrs/day at 1500 rpm.  Enough for the chlorinator to be able to do it's thing.  I'll be reducing these hours to see if I can get away with less.  
* My solar circuit is set to 1850 and that kicks in whenever the sun is out.
* I was letting my spa bypass do it's thing during the day but am going to turn it off and schedule the spa to run for 60 mins each day (to hopefully reduce acid demand).
* I have a waterfall that runs for 10 mins just to clear it out, but otherwise run it manually.
* I also currently run my cleaner manually but once I finish my REM conversion and hook up the 3hp relays to my Pi stack this will go on a schedule, too.  

![IMG_1739](https://user-images.githubusercontent.com/1646840/181114285-62346ca9-cba2-4807-b22c-75b6f36a28c2.jpg)
